cloud/filestore/libs/daemon/vhost/config_initializer.cpp

```diff
@@ -38,6 +38,10 @@ void TConfigInitializerVhost::InitAppConfig()
 
     VhostServiceConfig = std::make_shared<TVhostServiceConfig>(
         AppConfig.GetVhostServiceConfig());
+
+    if (Options->DiagnosticsConfig) {
```

Comment:
why do you need this ? 
DiagnostiConfigs already initialized in the base class by calling `TBootstrapCommon::InitCommonConfigs`